CONTRIBUTING.md

```diff
@@ -1,38 +1,38 @@
-# Libuavcan v1.0 contributor guidelines
+# Libcyphal v1.0 contributor guidelines
 
-v1.0 is a complete rewrite of libuavcan with the following, fundamental changes from v0:
+v1.0 is a complete rewrite of libcyphal with the following, fundamental changes from v0:
 
-1. libuavcan v1 is based on the [UAVCAN v1 specification](https://new.uavcan.org).
-2. libuavcan v1 requires C++11 or greater.
-3. libuavcan v1 favors idiomatic C++ over defining its own utilities and helpers.
-4. libuavcan v1 is a header-only library.
-5. libuavcan v1 holds itself to a higher quality standard and is designed for integration into high-integrity, embedded applications.
+1. libcyphal v1 is based on the [Cyphal v1 specification](https://new.opencyphal.org).
+2. libcyphal v1 requires C++11 or greater.
+3. libcyphal v1 favors idiomatic C++ over defining its own utilities and helpers.
+4. libcyphal v1 is a header-only library.
+5. libcyphal v1 holds itself to a higher quality standard and is designed for integration into high-integrity, embedded applications.
 
 ## About the Blue Sky effort
 
-You should never rewrite a codebase from scratch. It's a terrible idea. We're rewriting libuavcan from scratch. We're terrible developers.
+You should never rewrite a codebase from scratch. It's a terrible idea. We're rewriting libcyphal from scratch. We're terrible developers.
 
 Sorry.
 
-The reason we opted for a complete rewrite is that so much is changing. Between the updates to the specification, the abandonment of C++98 support, switching to header-only, removing the drivers from the main repository, adding CAN FD support, etc, etc, etc; it was obvious that we'd be rewriting everything anyway. The good news is that v0 exists, is fully supported, and will be liberally copy-and-pasted from as this makes sense for v1. What we don't have is any git history tracing from v0 to v1 since this would be deceiving. Futhermore, the unit tests in uavcan v0 are a bit of a mess so we'll be writing them in a way that is more sustainable.
+The reason we opted for a complete rewrite is that so much is changing. Between the updates to the specification, the abandonment of C++98 support, switching to header-only, removing the drivers from the main repository, adding CAN FD support, etc, etc, etc; it was obvious that we'd be rewriting everything anyway. The good news is that v0 exists, is fully supported, and will be liberally copy-and-pasted from as this makes sense for v1. What we don't have is any git history tracing from v0 to v1 since this would be deceiving. Futhermore, the unit tests in libcyphal v0 are a bit of a mess so we'll be writing them in a way that is more sustainable.
```

Comment:
```suggestion
The reason we opted for a complete rewrite is that so much is changing. Between the updates to the specification, the abandonment of C++98 support, switching to header-only, removing the drivers from the main repository, adding CAN FD support, etc, etc, etc; it was obvious that we'd be rewriting everything anyway. The good news is that v0 exists, is fully supported, and will be liberally copy-and-pasted from as this makes sense for v1. What we don't have is any git history tracing from v0 to v1 since this would be deceiving. Furthermore, the unit tests in v0 are a bit of a mess so we'll be writing them in a way that is more sustainable.
```